README.md

```diff
@@ -5,13 +5,11 @@ Complete any two out of the three challenges. See below for details of each chal
 ### Javascript Fluency Assignment
 
 #### Challenge - 1
-> In `src/assignment.mjs`, implement the `sumOfNumbers` function. Calculate and return the sum of the numbers in the given array. If you did Challenge - 1, remove the comment in the line just after the `sumOfNumbers` function, otherwise leave the comment.
+> In `src/assignment.js`, implement the `sumOfNumbersTo` function. Calculate and return the sum of the numbers from 1 to destination. If you did Challenge - 1, remove the comment in the line just after the `sumOfNumbersTo` function, otherwise leave the comment.
 
 #### Challenge - 2
-> In `src/assignment.mjs`, implement the `countEvenNumbers` function. Determine and return the count of all even numbers in the given array. If you did Challenge - 2, remove the comment in the line just after the `countEvenNumbers` function, otherwise leave the comment.
+> In `src/assignment.js`, implement the `countEvenNumbersWithin` function. Determine and return the count, sum, arrayOfEvenNumbers from 1 to destination. If you did Challenge - 2, remove the comment in the line just after the `countEvenNumbersWithin` function, otherwise leave the comment.
```

Comment:
This phrase " return the count, sum, arrayOfEvenNumbers from 1 to destination" is not clear around "count, sum, arrayOfEvenNumbers"